resources/scripts/components/server/schedules/TaskDetailsModal.tsx

```diff
@@ -91,7 +91,7 @@ const TaskDetailsForm = ({ isEditingTask }: { isEditingTask: boolean }) => {
                             <Label>Ignored Files</Label>
                             <FormikFieldWrapper
                                 name={'payload'}
-                                description={'Optional. Include the files and folders to be excluded in this backup. By default, the contents of your .pteroignore file will be used.'}
+                                description={'Optional. Include the files and folders to be excluded in this backup. By default, the contents of your .pteroignore file will be used.\nIf you have reached your backup limit, the oldest backup will be rotated.'}
```

Comment:
Does the `\n` work correctly here? I thought that would give you a string literal `\n` in the output, and not an actual newline? I don't even think it needs a newline honestly.

> Optional. Include the files and folders to be excluded in this backup. By default, the contents of your .pteroignore file will be used, and old backups will be automatically removed once your limit is reached.